website/client/components/chat/copyAsTodoModal.vue

```diff
@@ -1,11 +1,11 @@
 <template lang="pug">
   b-modal#copyAsTodo(:title="$t('copyMessageAsToDo')", :hide-footer="true", size='md')
     .form-group
-      input.form-control(type='text', v-model='task.text')
+      input.form-control(type='text', v-model='task.text', readonly)
```

Comment:
I think has been a mistake, the controls inside `copyAsTodoModal` should be always editable, people want and probably will edit the text/notes before saving it